docs/tasks/tls/certificate-rotation.md

```diff
@@ -0,0 +1,80 @@
+---
+approvers:
+- jcbsmpsn
+- mikedanese
+title: Certificate Rotation
+---
+
+{% capture overview %}
+This page shows how to enable and configure certificate rotation for the kubelet.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* Kubernetes version 1.8.0 or later is required
+
+* Encryption at rest is beta in 1.8.0 which means it may change without notice.
```

Comment:
Seems encryption is not strongly tied to this (rotation) topic?

Reply:
Oops, conflated two features I'm thinking about. Changed to "Kubelet certificate rotation"